### PR TITLE
[ASSIST-349]: Ensure Complete Mocked up Data in Redshift for Student, Enrollment, Program and new Student Performance table, and subpanels present in the App 

### PR DESCRIPTION
**Pull request by Orrison**
https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/349/

This requires seeded data for the other 3 ADM tables, which will be coming in an additional PR. But should work locally without.

### CODE REVIEW

File: app-modules/assist-data-model/database/migrations/sis/2023_07_17_200449_create_students_table.php [hunk 1]
```diff
@@ -18,26 +18,27 @@ public function up(): void
...
-                // TODO: Should dual be a boolean?
-                $table->string('dual')->nullable();
-                // TODO: Should ferpa be a boolean?
-                $table->string('ferpa')->nullable();
-                $table->float('gpa', 4, 3)->nullable();
-                $table->string('dfw')->nullable();
-                // TODO: Should firstgen be a boolean?
-                $table->string('firstgen')->nullable();
+                $table->boolean('dual')->nullable();
+                $table->boolean('ferpa')->nullable();
```
Comment (dgoetzit) [inline app-modules/assist-data-model/database/migrations/sis/2023_07_17_200449_create_students_table.php:33]: Should the `nullable` `boolean` fields instead be defaulted to false?

**Review by dgoetzit — DISMISSED** [review]
Just a couple of questions on my end, but this looks good!

File: app-modules/assist-data-model/database/factories/EnrollmentFactory.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * @extends Factory<Enrollment>
+ */
+class EnrollmentFactory extends Factory
+{
+    public function definition(): array
+    {
+        return [
+            'sisid' => $this->faker->randomNumber(9),
```
[the remark this answers is not in the document]
Comment (Orrison) [reply app-modules/assist-data-model/database/factories/EnrollmentFactory.php:16]: Yeah I can, but way may never get other ids

File: app-modules/assist-data-model/database/migrations/sis/2023_07_17_200449_create_students_table.php (hunk 1 above)
Comment (Orrison) [reply app-modules/assist-data-model/database/migrations/sis/2023_07_17_200449_create_students_table.php]: These actually don't make sense to be nullable, changed.

**Review by danharrin — APPROVED** [review]

**Pull request merged**